### ISSUE TITLE
comfyanonymous/flux_dev_scaled_fp8_test with flux-redux gives black images.

### ISSUE THREAD

**Issue by Dunc4n1dah0** [comfyanonymous/ComfyUI#5849]
### Expected Behavior

Make normal image.

### Actual Behavior

Make black image.

### Steps to Reproduce

1. use flux redux
2. use comfyanonymous/flux_dev_scaled_fp8_test model

I use normal nothing-special workflow with flux-redux. As soon as I switch diffusion model to downloaded from hf comfyanonymous/flux_dev_scaled_fp8_test from regular flux-dev image generated becomes black solid. Also GGUF model works ok. Again, issue shows only when using flux-redux. When I bypass group with redux nodes, image is correct.

![Screenshot 2024-11-30 154342](https://github.com/user-attachments/assets/6725a205-0603-4a4f-adfb-090b3bd260b6)
![Screenshot 2024-11-30 154415](https://github.com/user-attachments/assets/b59653a5-d5ae-4bbb-9baf-840cc700e5fd)

### Debug Logs

```powershell
Nothing unusual here, image generation finishes. It's just black.
```

**Comment by petercham** [comment]
![image](https://github.com/user-attachments/assets/0a2a4daa-4a19-484e-83f7-2bab76f30bd0)
I had a similar situation, first running it with Redux, then switching to the most basic dev raw map would only return a black map too

**Comment by KevinLi2023** [comment]
I met the same question, have you settled this problem?

<img width="1574" alt="Image" src="https://github.com/user-attachments/assets/7160d1aa-ff99-4b37-9bd2-f5f689312abe" />

**Comment by Dunc4n1dah0** [comment]
@KevinLi2023 yes, my solution is to not use comfyanonymous/flux_dev_scaled_fp8_test model...
